<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shadow Simulation</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionShadowing_Period"/>
    <addaction name="actionModel_Adjustment"/>
    <addaction name="actionConfigure_Shadows"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionShadowing_Period">
   <property name="text">
    <string>Shadowing Period</string>
   </property>
  </action>
  <action name="actionConfigure_Shadows">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Configure Shadows</string>
   </property>
  </action>
  <action name="actionModel_Adjustment">
   <property name="text">
    <string>Model Adjustment</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
